### ISSUE TITLE
Make sure files submitted to METADATA have unix line endings

### ISSUE THREAD

**Issue by tkelman** [attobot/attobot#25]
See https://github.com/JuliaLang/METADATA.jl/pull/8955 - we could possibly also enforce / fix this there with some git configury, but in that particular case it looks like attobot copied the files as-is with the same line endings as REQUIRE from the package repo.

**Comment by aytekinar** [comment]
How about adding

```
# Set the default behavior, in case people don't have core.autocrlf set.
* text eol=lf
```

to `.gitattributes` in `METADATA.jl`, following the example [here](https://help.github.com/articles/dealing-with-line-endings/)? Or, am I too naive?

```
 ~/dummy  git init; echo "* text eol=lf" > .gitattributes; git add .gitattributes ; git commit -m "Enforce LF"; echo "some line\nsome other line" > REQUIRE ; unix2dos REQUIRE ; git add REQUIRE; git commit -m "Add CRLF file"
Initialized empty Git repository in /home/aytekin/dummy/.git/
[master (root-commit) a3511f3] Enforce LF
 1 file changed, 1 insertion(+)
 create mode 100644 .gitattributes
unix2dos: converting file REQUIRE to DOS format...
warning: CRLF will be replaced by LF in REQUIRE.
The file will have its original line endings in your working directory.
[master a2801e8] Add CRLF file
 1 file changed, 1 insertion(+)
 create mode 100644 REQUIRE
```

**Comment by tkelman** [comment]
Would probably help. Dunno yet whether it would outright solve the problem, but couldn't hurt.

**Comment by simonbyrne** [comment]
Fixed by d0495c7d3774649011431cfd78cd45e9287b26bb
